deepspeed/ops/adam/cpu_adam.py

```diff
@@ -16,7 +16,8 @@ def __init__(self,
                         0.999),
                  eps=1e-8,
                  weight_decay=0,
-                 amsgrad=False):
+                 amsgrad=False,
+                 adamw_mode=True):

```

Comment:
Can we add a docstring to this class describing when and how it's used? I believe we only want to use it combined with zero-offload but we are getting users who are trying to use it solo which isn't supported.